samples/NativeLibrary/Class1.cs

```diff
@@ -51,5 +52,39 @@ public static IntPtr sumstring(IntPtr first, IntPtr second)
             // Return pointer
             return sumPointer;
         }
+
+        [UnmanagedCallersOnly(EntryPoint = "mergestrings")]
+        public static void MergeStrings(IntPtr oput, IntPtr strArr, int count)
```

Comment:
This example is using unsecure pattern. The buffer size needs to be passed together with the buffer pointer, and the function needs to handle the situation where the buffer passed in is not large enough.

It would make the sample a lot more complex. I am not sure whether having this sample is worth it.

Reply:
Sorry about that, completely missed that 🤦‍♂️ 

I thought it may have been of use to show how the native library could be used to manipulate existing memory allocation where strings are stored. 
I'm personally using this for creating extensions for a game server (that calls a function passing as parameters a memory location with data and a memory location where 'output'  has to be stored) . 
In case i'll just remove it